### DISCUSSION TITLE
Compiling Fortran Code: Prerequesites for Mac User

### DISCUSSION THREAD

**Discussion by schwemro** [phydrus/phydrus#29] [Compiling Fortran Code] [answered]
Dear Developers,

it would be great if you could extend the README by some prerequesites for mac users. A successful compilation of HYDRUS requires that a gfortran compiler is installed.

I followed these instructions [installing a gfortran compiler](http://hpc.sourceforge.net/).

Best,
Robin

**Comment by raoulcollenteur** [comment]
Hi Robin! Thanks for your comment. Did you try to use the executable from the GitHub? Any feedback on the steps to compile would be helpful here (also for other users :))

We plan to create a section on the documentation website on how to install and compile the Fortran code, unless this thing with compilation on Conda is going to work!

Cheers, Raoul

**Comment by schwemro** [comment, marked as answer]
Yes, but unfortunately the exectuable does not work unless I install the gfortran compiler (see link in my initial comment). Having installed `xcode-select` is not suffcient. After installing the compiler I had to recompile hydrus with the `make` command.

Robin
